
public class TryWithMultipleCatch {

	public static void main(String[] args) {
//		case 1
		
//		try {
//			
//			System.out.println(10/0);
//			
//		}catch(java.lang.ArithmeticException e) {
//			System.out.println("Arithmetic Exception ");
//		}		
//		catch (Exception e) {
//			
//			
//		}
	 
		
		
		// case 2:
		
		try {
			System.out.println(10/0);
		}
		
		catch (Exception e) {
			
		}
		catch (ArithmeticException e) {
			
		}
	
		/**
		 * CHILD PARENT RELATION
		 * */
		
	}
	
	

}
